crates/re_viewer/src/screenshotter.rs

```diff
@@ -0,0 +1,82 @@
+//! Screenshotting not implemented on web yet because we
+//! haven't implemented "copy image to clipboard" there.
+
+/// Helper for screenshotting the entire app
+#[derive(Default)]
+pub struct Screenshotter {
+    #[cfg(not(target_arch = "wasm32"))]
+    countdown: Option<usize>,
+
+    #[cfg(not(target_arch = "wasm32"))]
+    target_path: Option<std::path::PathBuf>,
+}
+
+#[cfg(not(target_arch = "wasm32"))]
+impl Screenshotter {
+    /// Used for generating screenshots in dev builds.
+    pub fn screenshot_to_path_then_quit(&mut self, path: std::path::PathBuf) {
+        self.request_screenshot();
+        self.target_path = Some(path);
```

Comment:
It's likely fine in practice, but this feels somewhat racy when it comes to repeated invocations. Not that repeated invocations are expected or needed, but maybe `request_screenshot` should warn if `self.countdown.is_some()`.

Reply:
I've added some clarifying comments and an assert, but I don't want to over-engineer this too much